src/scheduler.ts

```diff
@@ -66,10 +67,14 @@ class DefaultTaskScheduler extends TaskScheduler {
   #maximumUsage: number;
   #onAvailableListeners: ((item: PiscinaWorker) => void)[];
 
-  constructor (maximumUsage: number) {
+  constructor (maximumUsage: number = os.availableParallelism()) {
```

Comment:
Although we do not have the matrix anymore, we still _officially_ support v16; so feel free to do a check over the Node version running and only get the value if the `os.availableParallelism` is available.

Reply:
So in case the version is less than 16, what should this default to? There is `os.cpus()` that almost does the same, do we default to the length of that, or some other number?